mscore/scoreaccessibility.cpp

```diff
@@ -352,8 +352,7 @@ void ScoreAccessibility::updateAccessibilityInfo()
       if (!w)
             return;
 
-      if (w->score()->accessibleMessage().isEmpty())
-            currentInfoChanged();
+      currentInfoChanged();
```

Comment:
The `accessibleMessage` string is [cleared](https://github.com/musescore/MuseScore/blob/a3e23d6e63c3feff1058e6cf3616586785318af5/mscore/scoreaccessibility.cpp#L98) in the function `AccessibleScoreView::text()`, but that function only gets called if there is a screen reader running. If there is no screen reader then `accessibleMessage` would never be emptied after it was first set, hence `currentInfoChanged()` was never called to set the status bar text.